app/server/index.js

```diff
@@ -1,7 +1,36 @@
+import csv from 'csvtojson';
+import path from 'path';
 import { User, Product } from './models';
 import * as config from './config/config.json';
+import { DirWatcher } from './dirwatcher/dirwatcher';
+import { Importer } from './importer/importer';
 
 console.log('Application name is: ', config.appName);
 
 const Bob = new User();
 const Bread = new Product();
+
+const importer = new Importer();
+const pathToData = path.resolve(__dirname, 'data');
+
+importer.import(pathToData)
+    .then((path, data) => {
+        return new Promise((resolve, reject) => {
```

Comment:
To avoid callback hell, try to move the function outside of `then()` as the other examples on previous comments.